widget/view.controller.js

```diff
@@ -147,9 +300,491 @@
       });
     }
 
+    function moveVersionControlNext() {
+      WizardHandler.wizard('timSolutionpackWizard').next();
+    }
+
+    function _processHealthyConnectors(tabIndex, healthyConnectors) {
```

Comment:
please move this to service code --> Configingestionservice

function name needs to be changed to "ConfigureDataIngestion"

Reply:
please check if data ingestion can be configured using API if not then please create a mantis to support it